packages/context-menu/src/styles/vaadin-context-menu-overlay-styles.d.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * @license
+ * Copyright (c) 2000 - 2025 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import type { CSSResult } from 'lit';
+
+export declare const contextMenuOverlayStyles: CSSResult;
```

Comment:
We probably don't need `declare` here:

```suggestion
export const contextMenuOverlayStyles: CSSResult;
```